sbin/create_course.sh

```diff
@@ -14,56 +14,56 @@ fi
 
 CONF_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/../config
 
-SUBMITTY_REPOSITORY_DIR=$(jq -r '.submitty_repository' ${CONF_DIR}/submitty.json)
-SUBMITTY_INSTALL_DIR=$(jq -r '.submitty_install_dir' ${CONF_DIR}/submitty.json)
-SUBMITTY_DATA_DIR=$(jq -r '.submitty_data_dir' ${CONF_DIR}/submitty.json)
-SUBMISSION_URL=$(jq -r '.submission_url' ${CONF_DIR}/submitty.json)
+SUBMITTY_REPOSITORY_DIR=$(jq -r '.submitty_repository' "${CONF_DIR}/submitty.json")
+SUBMITTY_INSTALL_DIR=$(jq -r '.submitty_install_dir' "${CONF_DIR}/submitty.json")
+SUBMITTY_DATA_DIR=$(jq -r '.submitty_data_dir' "${CONF_DIR}/submitty.json")
+SUBMISSION_URL=$(jq -r '.submission_url' "${CONF_DIR}/submitty.json")
 
-PHP_USER=$(jq -r '.php_user' ${CONF_DIR}/submitty_users.json)
-DAEMON_USER=$(jq -r '.daemon_user' ${CONF_DIR}/submitty_users.json)
-CGI_USER=$(jq -r '.cgi_user' ${CONF_DIR}/submitty_users.json)
+PHP_USER=$(jq -r '.php_user' "${CONF_DIR}/submitty_users.json")
+DAEMON_USER=$(jq -r '.daemon_user' "${CONF_DIR}/submitty_users.json")
+CGI_USER=$(jq -r '.cgi_user' "${CONF_DIR}/submitty_users.json")
 
-COURSE_BUILDERS_GROUP=$(jq -r '.course_builders_group' ${CONF_DIR}/submitty_users.json)
+COURSE_BUILDERS_GROUP=$(jq -r '.course_builders_group' "${CONF_DIR}/submitty_users.json")
 
-DATABASE_HOST=$(jq -r '.database_host' ${CONF_DIR}/database.json)
-DATABASE_PORT=$(jq -r '.database_port' ${CONF_DIR}/database.json)
-DATABASE_USER=$(jq -r '.database_user' ${CONF_DIR}/database.json)
-DATABASE_PASS=$(jq -r '.database_password' ${CONF_DIR}/database.json)
+DATABASE_HOST=$(jq -r '.database_host' "${CONF_DIR}/database.json")
+DATABASE_PORT=$(jq -r '.database_port' "${CONF_DIR}/database.json")
+DATABASE_USER=$(jq -r '.database_user' "${CONF_DIR}/database.json")
+DATABASE_PASS=$(jq -r '.database_password' "${CONF_DIR}/database.json")
 
 ########################################################################################################################
 ########################################################################################################################
 
 CONN_STRING="-h ${DATABASE_HOST} -U ${DATABASE_USER}"
-if [ -d ${DATABASE_HOST} ]; then
+if [ -d "${DATABASE_HOST}" ]; then
     CONN_EXTRA="${CONN_STRING} -p ${DATABASE_PORT}"
 fi
 
 # Check that Submitty Master DB exists.
 PGPASSWORD=${DATABASE_PASS} psql ${CONN_STRING} -lqt | cut -d \| -f 1 | grep -qw submitty
```

Comment:
Considering quoting the `${DATABASE_PASS}` ([SC2086](https://www.shellcheck.net/wiki/SC2086))?

```suggestion
PGPASSWORD="${DATABASE_PASS}" psql ${CONN_STRING} -lqt | cut -d \| -f 1 | grep -qw submitty
```